src/components/ReminderForm.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { TwitterPicker } from 'react-color';
+
+import { Reminder } from '../lib/Reminder';
+import { getDateHourMinutes, formatDateFromTime } from '../helpers/hours';
+
+class ReminderForm extends React.Component {
```

Comment:
`import { Component }, React from 'react` creo que asi era la syntax, y podes hacer `extends Component` directamente

Reply:
for the sake of readability